stacks/ibm/application/istio/kustomization.yaml

```diff
@@ -2,9 +2,9 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 namespace: kubeflow
 resources:
-- ../../../../istio/istio/base
+- ../../../../istio/istio/base_v3
```

Comment:
Maybe not particularly relevant to this PR, but since some resources here are being updated to `base_v3`, should `stacks/ibm/application/istio-1-3-1-stack/kustomization.yaml` and `stacks/ibm/application/cluster-local-gateway-1-3-1/kustomization.yaml` now also reference their respective `base_v3` directories?

Reply:
Minor note: the equivalent of `istio-1-3-1-stack` is `istio-stack`. I updated the manifests to use `base_v3`.